docs/docs/concepts/persistence.md

```diff
@@ -147,24 +147,21 @@ In our example, the output of `get_state_history` will look like this:
 
 ### Replay
 
-It's also possible to play-back a prior graph execution. If we `invoking` a graph with a `thread_id` and a `checkpoint_id`, then we will *re-play* the graph from a checkpoint that corresponds to the `checkpoint_id`.
+It's also possible to play-back a prior graph execution. If we `invoke` a graph with a `thread_id` and a `checkpoint_id`, then we will *re-play* the previously executed steps BEFORE a checkpoint that corresponds to the `checkpoint_id`, and only execute the steps AFTER the checkpoint.
 
-* `thread_id` is simply the ID of a thread. This is always required.
-* `checkpoint_id` This identifier refers to a specific checkpoint within a thread. 
+* `thread_id` is the ID of a thread.
+* `checkpoint_id` is an identifier that refers to a specific checkpoint within a thread.
 
 You must pass these when invoking the graph as part of the `configurable` portion of the config:
 
 ```python
-# {"configurable": {"thread_id": "1"}}  # valid config
-# {"configurable": {"thread_id": "1", "checkpoint_id": "0c62ca34-ac19-445d-bbb0-5b4984975b2a"}}  # also valid config
-
-config = {"configurable": {"thread_id": "1"}}
+config = {"configurable": {"thread_id": "1", "checkpoint_id": "0c62ca34-ac19-445d-bbb0-5b4984975b2a"}}
 graph.invoke(None, config=config)
 ```
 
-Importantly, LangGraph knows whether a particular checkpoint has been executed previously. If it has, LangGraph simply *re-plays* that particular step in the graph and does not re-execute the step. See this [how to guide on time-travel to learn more about replaying](../how-tos/human_in_the_loop/time-travel.ipynb).
+Importantly, LangGraph knows whether a particular step has been executed previously. If it has, LangGraph simply *re-plays* that particular step in the graph and does not re-execute the step, but only for the steps BEFORE the provided `checkpoint_id`. All of the steps AFTER `checkpoint_id` will be executed (i.e., a new fork), even if they have been executed previously. See this [how to guide on time-travel to learn more about replaying](../how-tos/human_in_the_loop/time-travel.ipynb).
```

Comment:
may be confusing w/ `before` and `after` about what the actual starting point is